.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "bracketSpacing": false,
+  "jsxBracketSameLine": true,
+  "singleQuote": true,
+  "tabWidth": 4,
+  "useTabs": true
+}
```

Comment:
I tried to pick settings that seem to match the (kind of) existing desired style. Do we have a definitive style guide for this anywhere, or a repo where we're already using Prettier with some sort of org-approved settings?

Reply:
Yes, we're using in [clay](https://github.com/liferay/clay.git). 

I guess [here](https://github.com/liferay/clay/blob/master/package.json#L15) and maybe [here](https://github.com/liferay/clay/blob/master/.eslintrc) could be a good place to look.